sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 - Added support for the `WebSockets` protocol via the `Azure::Core::Http::WebSockets::WebSocket` class.
 - Added support for HTTP proxy servers, both unauthenticated and with Plain authentication.
+- Added universal support for several TLS options:
+  - Added the ability to set the expected TLS root certificate for TLS connection (useful if a proxy server uses a TLS certificate that is not chained to a known root).
+  - Added the ability to enable TLS certificate revocation list checks (off by default)
+- Added LibCURL functionality for TLS functionality
+    - Allow TLS connection to succeed if CRL retrieval fails.
```

Comment:
Is it worth having this as a separate sub-bullet? Could it be fine as a note in brackets to the main bullet? Also, reading functionality twice is a bit awkward:
Something like:
```suggestion
- Added TLS functionality for LibCurl (allow TLS connection to succeed if CRL retrieval fails).
```

If you want to keep it, should the main bullet end in period or colon?